pagopa/io-app
# feat: [IOCOM-854] Add bottom sheet to display message contacts

## Short description
This PR adds the bottom sheet to display message contacts.

<details open><summary>Details</summary>
<p>

| bs |
| - |
| <img src="https://github.com/pagopa/io-app/assets/29163287/0c0b4d07-3126-4f40-aa82-b6bc8b0323d0" width="300"/> |

</p>
</details> 

## List of changes proposed in this pull request
- added useMessageContactsBottomSheet hook to display the bottom sheet
- updated locales
- updated tests

## How to test
Using `io-dev-api-server`, generate a message. Activate experimental ds and open a message. Tap on _contacts_ action and check that the bottom sheet is displayed correctly.
